SortApp/SortApp.Presentation/MainWindow.xaml

```diff
@@ -5,7 +5,7 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:SortApp.Presentation"
         mc:Ignorable="d"
-        Title="MainWindow" Height="350" Width="525">

```

Comment:
I don't see a need for this change. It could be reverted. Windows size and other stuff will be changed when we'll start to work with UI.